packages/front-end/pages/features/index.tsx

```diff
@@ -219,12 +239,18 @@ export default function FeaturesPage() {
                 let version = feature.revision?.version || 1;
                 if (isDraft) version++;
 
-                return (
+                return !(!feature.archived || showArchived) ? null : (
```

Comment:
If you do the filtering here, it will mess up pagination.  Imagine the first 19 features are archived.  That would mean page 1 would only show 1 feature.

Instead, I would do the filtering before doing the slice:

```tsx
<tbody>
  {sorted.filter(f=>!f.archived).slice(start, end).map((feature) => {
    ...
```
```

Reply:
Changed in recent commit